crates/wit-bindgen/src/lib.rs

```diff
@@ -125,6 +125,10 @@ pub struct Opts {
     /// Whether or not to emit `tracing` macro calls on function entry/exit.
     pub tracing: bool,
 
+    /// Whether or not `tracing` macro calls should included argument and
+    /// return values which contain dynamically-sized `list` values.
+    pub verbose_tracing: bool,
+
```

Comment:
Mind hooking this up as well to the macro input?

Reply:
Done!